scripts/priceBoundPSMDeployDai.ts

```diff
@@ -0,0 +1,118 @@
+import hre, { ethers } from 'hardhat';
+import NetworksForVerification from '@protocol/networksForVerification';
+import { PriceBoundPSM } from '@custom-types/contracts';
+import { expect } from 'chai';
+import { getAllContractAddresses } from './utils/loadContracts';
+import { NamedAddresses } from '@custom-types/types';
+
+const voltFloorPrice = '9000000000000000';
+const voltCeilingPrice = '10000000000000000';
+
+const daiReservesThreshold = ethers.utils.parseEther('10000000000');
+const mintLimitPerSecond = ethers.utils.parseEther('10000');
+const voltPSMBufferCap = ethers.utils.parseEther('10000000');
+const addressOne = '0x0000000000000000000000000000000000000001';
+
+async function deploy(contractAddresses: NamedAddresses) {
```

Comment:
let's not use this script file and instead use the model in the `proposals/dao/vip_x.ts`